tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -3350,8 +3337,10 @@ write_instructions(LL_Module *module)
 
         assert(p->next == NULL, "write_instructions(), bad next ptr", 0,
                ERR_Fatal);
-        write_no_depcheck_metadata(module, instrs);
-        write_llaccgroup_metadata(module, instrs);
+        write_llaccgroup_metadata(module, instrs,
+			LDST_HAS_METADATA, instrs->misc_metadata);
```

Comment:
Please use spaces instead of tabs for indentation, and left-align multi-line parameter lists.